doc/control-api.md

```diff
@@ -49,3 +49,9 @@ Here is the supported API:
 Introduced since `v2.2`.
 
 Return the jsonschema used by this APISIX instance.
+
+### GET /v1/server_info
```

Comment:
I think it would be better to put this to plugin's own section. So that the API in this page is builtin and will be provided whether you use the plugin or not.

Reply:
Fixed.